__init__.py

```diff
@@ -24,16 +24,16 @@ def name():
 def description():
     return "Edit toolbar for PostGIS topology primitives (ISO SQL/MM based)"
 def version():
-    return "0.2.2"
+    return "0.3.1"
```

Comment:
why 0.3.1 ? what's the point of changing this version ?

Reply:
None really. Just missed reverting it. Though I'd bump it to 0.3 since it's a 3 as in QGIS3.

Reply:
I'd bump to 1.0 as it _drops_ support for older qgis version (breaking change)